xcube/core/verify.py

```diff
@@ -81,9 +81,41 @@ def verify_cube(dataset: xr.Dataset) -> List[str]:
     if xy_var_names and time_var_name:
         _check_data_variables(dataset, xy_var_names, time_var_name, report)
 
+    if xy_var_names:
+        _check_coord_equidistance(dataset, xy_var_names[0], xy_var_names[0], report)
+        _check_coord_equidistance(dataset, xy_var_names[1], xy_var_names[1], report)
+
     return report
 
 
+def _check_coord_equidistance(dataset, coord_name, dim_name, report, rtol=None):
+    diff = dataset[coord_name].diff(dim=dim_name)
+    if not _check_equidistance_from_diff(diff, rtol=rtol):
+        report.append(f"Coordinate {coord_name} is not equidistant")
+
+    bnds_name = dataset.attrs.get('bounds', f'{coord_name}_bnds')
+
+    if bnds_name in dataset.coords:
+        diff = dataset[bnds_name].diff(dim=dim_name)
+        if not _check_equidistance_from_diff(diff[:, 0], rtol=rtol):
+            report.append(f"Coordinate {bnds_name} is not equidistant")
+        elif not _check_equidistance_from_diff(diff[:, 1], rtol=rtol):
+            report.append(f"Coordinate {bnds_name} is not equidistant")
+
+
+def _check_equidistance_from_diff(diff, rtol=None):
+    if rtol is None:
+        rtol = np.abs(np.divide(diff[0], 100.00)).values
+
+    # Check whether teh bounding box intersect with the anti-meridian.
+    ct_neg = diff.where(diff < 0).count().values
+
+    if ct_neg == 1:
+        return np.allclose(diff[0], diff.where(diff > 0, drop=True), rtol=rtol)
```

Comment:
```suggestion
        # <please explain, why you use diff.where(diff > 0, drop=True) here>
        return np.allclose(diff[0], diff.where(diff > 0, drop=True), rtol=rtol)
```

Reply:
Short question: At this stage, the verification excludes the anti-medirian intersecting bound from the verification. Is that OK? We could compute the distance by adding 360degrees to the difference. But that would be projection specific.

Reply:
Good point. We should at least detect anti-meridian crossing, and if we cross, skip equidistance checking for time being (will implement for 0.4). Use `is_lon_lat_dataset()` to distinguish from other CRS. 
In various places, I use the condition `lon_start > lon_end` to detect anti-meridian crossing for bboxes.

Reply:
see
* https://github.com/dcs4cop/xcube/blob/8ff6fcbe73a2145de48958d541c283d9583009f2/xcube/webapi/controllers/tiles.py#L262
* https://github.com/dcs4cop/xcube/blob/8ff6fcbe73a2145de48958d541c283d9583009f2/xcube/core/geom.py#L453
* https://github.com/dcs4cop/xcube/blob/520cd2cf7e3f4d2d121889064bdf3732863c244f/xcube/util/tilegrid.py#L214